lib/Driver/ToolChains.cpp

```diff
@@ -1318,121 +1337,23 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   return {"clang++", Arguments};
 }
 
-ToolChain::InvocationInfo
-toolchains::Windows::constructInvocation(const InterpretJobAction &job,
-                                         const JobContext &context) const {
-  InvocationInfo II = ToolChain::constructInvocation(job, context);
-
-  SmallString<128> runtimeLibraryPath;
-  getRuntimeLibraryPath(runtimeLibraryPath, context.Args, *this);
-
-  addPathEnvironmentVariableIfNeeded(II.ExtraEnvironment, "LD_LIBRARY_PATH",
-                                     ":", options::OPT_L, context.Args,
-                                     runtimeLibraryPath);
-  return II;
+std::string toolchains::Cygwin::getDefaultLinker() const {
+  // Cygwin uses the default BFD linker.
```

Comment:
Nitpick: can you add "even on ARM" so it's clear what's different from GenericUnix?